nnvm/python/nnvm/to_relay.py

```diff
@@ -0,0 +1,530 @@
+    axis = attrs.get_int('axis', 1)
+    epsilon = attrs.get_float('epsilon', 1e-05)
+    center = attrs.get_bool('center', True)
+    scale = attrs.get_bool('scale', True)
+
+    return op.nn.batch_norm(
+        data,
+        gamma,
+        beta,
+        moving_mean,
+        moving_view,
+        axis=axis,
+        epsilon=epsilon,
+        center=center,
+        scale=scale)[0]
+
+
+def _to_max_pool2d(children, attrs, odtype):
+    assert len(children) == 1
+    data = children[0]
+    pool_size = attrs.get_int_tuple('pool_size', (1, 1))
+    strides = attrs.get_int_tuple('strides', (1, 1))
+    padding = attrs.get_int_tuple('padding', (0, 0))
+    layout = attrs.get_int_tuple('layout', 'NCHW')
+    ceil_mode = attrs.get_int_tuple('ceil_mode', False)
+
+    return op.nn.max_pool2d(
+        data,
+        pool_size=pool_size,
+        strides=strides,
+        padding=padding,
+        layout=layout,
+        ceil_mode=ceil_mode)
+
+
+def _to_reshape(children, attrs, odtype):
+    data = children[0]
+    shape = attrs.get_int_list('shape')
+    return op.reshape(data, shape)
+
+
+def _to_transpose(children, attrs, odtype):
+    axes = attrs.get_int_list('axes', None)
+    return op.transpose(children[0], axes=axes)
+
+
+def _to_add(children, attrs, odtype):
+    if len(children) == 1:
+        left = children[0]
+        scalar = attrs.get_float('scalar')
+        right = relay.const(scalar, dtype=odtype)
+    else:
+        assert len(children) == 2
+        left = children[0]
+        right = children[1]
+
+    return op.add(left, right)
+
+
+def _to_subtract(children, attrs, odtype):
+    if len(children) == 1:
+        left = children[0]
+        scalar = attrs.get_float('scalar')
+        right = relay.const(scalar, dtype=odtype)
+    else:
+        assert len(children) == 2
+        left = children[0]
+        right = children[1]
+
+    return op.subtract(left, right)
+
+
+def _to_rsubtract(children, attrs, odtype):
+    if len(children) == 1:
+        left = children[0]
+        scalar = attrs.get_float('scalar')
+        right = relay.const(scalar, dtype=odtype)
+    else:
+        assert len(children) == 2
+        left = children[0]
+        right = children[1]
+
+    return op.subtract(right, left)
+
+
+def _to_multiply(children, attrs, odtype):
+    if len(children) == 1:
+        left = children[0]
+        scalar = attrs.get_float('scalar')
+        right = relay.const(scalar, dtype=odtype)
+    else:
+        assert len(children) == 2
+        left = children[0]
+        right = children[1]
+
+    return op.multiply(left, right)
+
+
+def _to_divide(children, attrs, odtype):
+    if len(children) == 1:
+        left = children[0]
+        scalar = attrs.get_float('scalar')
+        right = relay.const(scalar, dtype=odtype)
+    else:
+        assert len(children) == 2
+        left = children[0]
+        right = children[1]
+
+    return op.divide(left, right)
+
+
+def _to_rshift(children, attrs, odtype):
```

Comment:
consider summarizing function as ```_rbinop_scalar(new_op)``` as in https://github.com/dmlc/tvm/blob/master/python/tvm/relay/frontend/mxnet.py#L130